contracts/RelayHub.sol

```diff
@@ -368,8 +368,6 @@ contract RelayHub is IRelayHub {
         require(info.owner == msg.sender, "not owner");
         require(info.withdrawBlock > 0, "Withdrawal is not scheduled");
         require(info.withdrawBlock <= block.number, "Withdrawal is not due");
-        require(relayData[relayManager].manager != address(0), "Relay is not registered");
-        relayData[relayManager].penalized = true;
```

Comment:
the logic for `getRelayInfoForManagers` checks for unlock state, not withdraw, so this was moved (see line `362` on the right)